.github/requirements-dev.txt

```diff
@@ -25,7 +25,7 @@ patchelf==0.17.2.2 ; sys_platform == "linux" and platform_machine == "x86_64" \
     --hash=sha256:47b558db8f7dccc6262e930631991ecdd053724c1d8daf3df5ce03813438da10 \
     --hash=sha256:b54e79ceb444ec6a536a5dc2e8fc9c771ec6a1fa7d5f4dbb3dc0e5b8e5ff82e1 \
     --hash=sha256:e334ebb1c5aa9fc740fd95ebe449271899fe1e45a3eb0941300b304f7e3d1299
-    # via -r .github/requirements-dev.in
+    # via -r requirements-dev.in
```

Comment:
Nitpick: Why is `.github/` removed?

Reply:
Apparently my local pip-compile works slightly differently compared to GitHub Dependabot pip-compile. It is a comment anyway and doesn't affect the execution.